### PR TITLE
GIFLib moved into a subdir and bumped to 5.1.4 (left shared libs of 5.0.6 in place too)

### PR DESCRIPTION
**Pull request by jimklimov**

### CODE REVIEW

**Comment by pyhalov** [comment]
dependent packages:
e/efl
library/desktop/search/tracker

**Comment by pyhalov** [comment]
Need to rebuild efl and merge #2013  after integration

File: components/library/giflib/Makefile [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+$(BUILD_DIR_64)/.installed:     $(BUILD_OLD_DIR_64)/.installed
+
+
+build: $(BUILD_32_and_64)
+
+install: $(INSTALL_32_and_64)
+
+test: $(NO_TESTS)
+
+BUILD_PKG_DEPENDENCIES =    $(BUILD_TOOLS)
```
Comment (pyhalov) [inline components/library/giflib/Makefile]: Please,  remove BUILD_PKG_DEPENDENCIES and depend.mk lines, they are not used

Comment (jimklimov) [inline components/library/giflib/Makefile]: done

**Pull request merged**